sdk/cpp/tests/main.cpp.in

```diff
@@ -10,7 +10,7 @@ int main( int argc, char* argv[] )
 {
   // global setup...
   auto logger = spdlog::stdout_color_mt("ydk");
-  logger->set_level(spdlog::level::err);
+  logger->set_level(spdlog::level::debug);
```

Comment:
Setting to `debug` explodes Travis CI log, add will cause build failure if it exceed 4 MB limit.